src/main/java/calculator/Application.java

```diff
@@ -1,7 +1,12 @@
 package calculator;
 
+import calculator.calculator.CalculatorController;
+import calculator.global.CustomBeanFactory;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        CustomBeanFactory customBeanFactory = new CustomBeanFactory();
+        CalculatorController calculatorController = customBeanFactory.calculatorController();
+        calculatorController.run();
```

Comment:
개인적으론 재사용성이 없는 객체는 인스턴스화 하는 것을 선호하지는 않아요.
지역 변수, 인스턴스가 많아질 수록 관리 포인트가 늘어나고 예상치 못한 동작이 많아진다고 생각합니다.
```suggestion
        customBeanFactory.calculatorController().run();
```